src/main/java/sirius/kernel/nls/NLS.java

```diff
@@ -667,10 +663,16 @@ public static DateTimeFormatter getDateTimeFormat(String lang) {
      * @return a format initialized with the pattern described by the given language
      */
     public static DateTimeFormatter getDateTimeFormatWithoutSeconds(String lang) {
-        return dateTimeFormatters.computeIfAbsent(lang,
-                                                  l -> DateTimeFormatter.ofPattern(get(
-                                                          "NLS.patternDateTime.withoutSeconds",
-                                                          l)));
+        return safeGetDateTimeFormat(lang, dateTimeFormatters, "NLS.patternDateTime.withoutSeconds");
+    }
+
+    private static DateTimeFormatter safeGetDateTimeFormat(String lang,
+                                                           Map<String, DateTimeFormatter> formatters,
+                                                           String nlsProperty) {
+        if (Strings.isEmpty(lang)) {
+            lang = getCurrentLang();
```

Comment:
assigning to parameters is commonly not the preferred choice - we could just return here...